packages/es-client/tests/test-es-indexer.js

```diff
@@ -415,26 +353,13 @@ test.serial('delete a provider record', async (t) => {
   );
 });
 
-// This needs to be serial because it is stubbing aws.sfn's responses
-test.serial('reingest a granule', async (t) => {
```

Comment:
I know you didn't necessarily write these test statements but I think all the tests statements in this file can be improved because it doesn't really make any clear assertion or claim of what to expect or what it is trying to test.

Reply:
While I probably agree, I'd like to avoid making changes to "improve" the test code beyond just moving them and getting them to pass. The only test changes I made were to make the tests pass after the move while eliminating dependencies on the API package. Otherwise this PR could become even more unwieldy. 

I realize that's confusing since I deleted this test. I only deleted this test because a) it no longer seemed necessary and b) it relied on code from the API package (the granules model) which would be bad because we can't make `es-clienta depend on `api`, but since it wasn't a necessary test anymore, the easiest solution was just to remove it.